src/scss/objects/_links.scss

```
@@ -37,6 +37,12 @@
 
     .o-link--noDecoration {
         text-decoration: none;
+
+        // for accessibility reasons we are leaving
+        // text-decoration on hover and focus
+        &:hover, &:focus {
```

NIT: can you wrap the rules onto separate lines, please?

```suggestion
        &:hover,
        &:focus {
```